autorun_plot.sh

```diff
@@ -15,8 +15,8 @@ echo '<a href="https://github.com/facebookresearch/nevergrad/blob/master/nevergr
 for xp in $listxp
 do
     echo working on $xp
-    echo "<br> $xp </br><p>" >> allxps/list.html
```

Comment:
can you please move those files to dfoptim instead? Then I'll create a new release including the recent changes, and **before** I may break everything with the new parametrization framework